wingui/pdcscrn.c

```diff
@@ -1,7 +1,9 @@
 /* Public Domain Curses */
 
 #include "pdcwin.h"
+#include <shellapi.h>
```

Comment:
What do we need this additional include for? I'm asking as this is not part of the PR issue text.

Reply:
Similar issue to above,  except this time,  `ExtractIcon()` is missing in lean/mean mode.